packages/expansion-hub/src/ExpansionHub.ts

```diff
@@ -57,12 +58,37 @@ export interface ExpansionHub extends RevHub {
     getFTDIResetControl(): Promise<boolean>;
 
     // DIO
-    setDigitalSingleOutput(dioPin: number, value?: boolean): Promise<void>;
-    setDigitalAllOutputs(bitPackedField: number): Promise<void>;
-    setDigitalDirection(dioPin: number, direction: DIODirection): Promise<void>;
-    getDigitalDirection(dioPin: number): Promise<DIODirection>;
-    getDigitalSingleInput(dioPin: number): Promise<boolean>;
-    getDigitalAllInputs(): Promise<number>;
+    /**
+     * Set the output state of a digital pin.
+     * @param digitalChannel
+     * @param value whether the pin should be High or Low
+     */
+    setDigitalOutput(digitalChannel: number, value: DigitalState): Promise<void>;
+    setAllDigitalOutputs(bitPackedField: number): Promise<void>;
```

Comment:
Needs a doc comment

Reply:
Do we know how this method reacts if any pins are set to input? Will it skip the input pins, or will it throw a nack?